sample.env

```diff
@@ -87,9 +87,12 @@ MAPBOX_INIT_LONGITUDE="-104.9903"
 MAPBOX_INIT_ZOOM="10"
 
 # Firmware Manager
-BLOB_STORAGE_PROVIDER=GCP
-BLOB_STORAGE_BUCKET=
+BLOB_STORAGE_PROVIDER=DOCKER

```

Comment:
Is this now either GCP or Docker? Should we indicate that somewhere?

Reply:
This has been modified to make the supported options more apparent.